runtime/layers/fpm/bootstrap

```diff
@@ -25,8 +25,7 @@ if (getenv('BREF_DOWNLOAD_VENDOR')) {
     require $appRoot . '/vendor/autoload.php';
 }
 
-// Get a LambdaRuntime and disable timeout exceptions.
-$lambdaRuntime = LambdaRuntime::fromEnvironmentVariable(-1);
+$lambdaRuntime = LambdaRuntime::fromEnvironmentVariable();
```

Comment:
In my serverless.yaml, I have 3 or 4 lambda functions. Ie, web, console, sqs-worker. I define my environment variables once. If I define `BREF_FEATURE_TIMEOUT=1`, then even the FPM (root) process will enable timeout. 

That is not something we want.

Reply:
Good point, I pushed something to prevent this thanks!